packages/liveblocks-devtools/turbo.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "dev:firefox": {
+      "dependsOn": ["^build"],
+      "cache": false
+    },
+    "build:firefox": {
+      "dependsOn": ["^build"]
+    }
+  }
+}
```

Comment:
…instead, you can "extend" the global version, and make these local, so this config lives in the workspace where these commands make sense.